src/admin-views/filter_bar/upsell.php

```diff
@@ -6,28 +6,29 @@
  * @since 5.14.0
  * @since TBD Drastically simplified the HTML, changed classes to use new admin styles
  */
+_deprecated_file( __FILE__, 'TBD', 'src/admin-views/settings/upsells/filter_bar.php' );
 $main = Tribe__Events__Main::instance();
 ?>
 <div class="tec-settings__upsell">
+	<div class="tec-settings__upsell-header">
+		<img
+			src="<?php echo esc_url( tribe_resource_url( 'icons/filterbar.svg', false, null, $main ) ); ?>"
+			class="tec-settings__upsell-logo"
+			role="presentation
```

Comment:
Missing a closing quote.

```suggestion
			role="presentation"
```